crates/dbs-utils/src/net/mod.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the THIRD-PARTY file.
```

Comment:
The license file `THIRD-PARTY` is missing from the dbs-utils crate. There are several copies of `THIRD-PARTY` file within the project, so we should move it to the top directory and rename it as `LICENSE-BSD-Google`

Reply:
I have opened issue #117 to track this issue and will deal with it in the next pr.